internal/recert/recert.go

```diff
@@ -59,6 +71,29 @@ func FormatRecertProxyFromSeedReconfigProxy(proxy, statusProxy *seedreconfig.Pro
 	)
 }
 
+func SetRecertTrustedCaBundleFromSeedReconfigAdditionaTrustBundle(recertConfig *RecertConfig, additionalTrustBundle seedreconfig.AdditionalTrustBundle) error {
+	if additionalTrustBundle.UserCaBundle != "" {
+		recertConfig.UserCaBundle = additionalTrustBundle.UserCaBundle
+	}
+
+	if additionalTrustBundle.ProxyConfigmapName != "" {
+		if additionalTrustBundle.ProxyConfigmapBundle == "" {
+			return fmt.Errorf("proxy configmap bundle is empty while proxy configmap name is set")
+		}
+
+		switch additionalTrustBundle.ProxyConfigmapName {
+		case common.ClusterAdditionalTrustBundleName:
+			recertConfig.ProxyTrustedCaBundle = fmt.Sprintf("%s:", common.ClusterAdditionalTrustBundleName)
+		case "":
```

Comment:
Hmm, this case will not be reachable or i miss something ? You actually validate in line 79 that additionalTrustBundle.ProxyConfigmapName is not empty string?

Reply:
True, fixed